khal/api.py

```diff
@@ -0,0 +1 @@
+from .ui.colors import register_color_theme  # noqa
```

Comment:
You can use the following to indicate that this module explicitly exports a function:

```py
__all__ = ["register_color_theme"]
```

Mainly, it'll prevent the "unused import" warning entirely while conveying some meaning to casual readers.

Reply:
Thanks, implemented